actionview/test/template/form_helper/form_with_test.rb

```diff
@@ -381,6 +381,26 @@ def test_form_with
     assert_dom_equal expected, output_buffer
   end
 
+  def test_form_with_with_persisted_to_model
+    post_form = OpenStruct.new(to_model: @post)
```

Comment:
```suggestion
    post_form = Struct.new(to_model: @post)
```
These can be simple structs instead of open ones.